IoT Edge anomaly detection tutorial/iot-workshop-deployment-template.json

```diff
@@ -64,7 +64,7 @@
             "restartPolicy": "always",
             "settings": {
               "image": "__REGISTRY_IMAGE_LOCATION",
-			        "createOptions": "{\"HostConfig\":{\"Binds\":[\"/data/misc/camera:/app/vam_model_folder\"],\"NetworkMode\":\"host\"},\"NetworkingConfig\":{\"EndpointsConfig\":{\"host\":{}}},\"WorkingDir\":\"/var/azureml-app\"}"
+              "createOptions": "{\"HostConfig\":{\"Binds\":[]},\"WorkingDir\":\"/var/azureml-app\"}"
```

Comment:
If you're getting rid of the binds anyways (which I just checked does work), I think you can get rid of the whole HostConfig leaving you just with the WorkingDir setting.

Reply:
Actually, I just deleted all the createOptions and it seems to be fine...

Reply:
Updated. Thanks.